google/resource_compute_shared_vpc_service_project.go

```diff
@@ -85,6 +121,38 @@ func resourceComputeSharedVpcServiceProjectRead(d *schema.ResourceData, meta int
 		return nil
 	}
 
+	// We don't want to iterate through all subnetworks, checking to see which
+	// ones have k8s access.  In that sense, this resource isn't 'authoritative'.
```

Comment:
Why not?

Reply:
Haven't thought about it much - it's just that it's not the way that the buttons work on the cloud console.  Do you think we ought to?  That would be fine by me!

Reply:
How do the buttons work? It seems to me like you just click on the ones you want, and leave the ones you don't want unselected. So I guess you could argue either way that that's how the buttons work. I was thinking that if we did iterate through all of them, then import would work, right? You'd have to add a migration fn so we don't end up with another situation where people upgraded and saw diffs like in app engine, but that's not terrible.